.github/workflows/actionlint.yml

```
@@ -74,13 +74,13 @@ jobs:
       security-events: write
     steps:
       - name: Download SARIF file
-        uses: actions/download-artifact@v4
+        uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4
         with:
           name: results.sarif
           path: results.sarif
 
       - name: Upload SARIF file
-        uses: github/codeql-action/upload-sarif@v3
+        uses: github/codeql-action/upload-sarif@3fb25ab90399ba0ad65c205b2c992bc352d8718a # v3
```

```suggestion
        uses: github/codeql-action/upload-sarif@f09c1c0a94de965c15400f5634aa42fac8fb8f88 # v3
```

This looks like https://github.com/stacklok/frizbee/issues/206. `3fb25ab` is the hash of the tag, not the commit that the tag points to:

```console
$ git show 3fb25ab90399ba0ad65c205b2c992bc352d8718a
tag v3
Tagger: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
Date:   Wed Nov 20 14:31:01 2024 +0000

v3

commit f09c1c0a94de965c15400f5634aa42fac8fb8f88 (tag: v3.27.5, tag: v3, origin/releases/v3)
Merge: ea9e4e379 67b73eaba
Author: Marco Gario <marcogario@github.com>
Date:   Wed Nov 20 15:29:45 2024 +0100

    Merge pull request #2616 from github/update-v3.27.5-a6c8729a5

    Merge main into releases/v3
```

GitHub Actions can resolve that correctly, but the tag's hash is not very visible so I think we should avoid that.

---

(Committed via #213)